src/Akeneo/Crowdin/TranslatedProgressSelector.php

```diff
@@ -43,36 +60,102 @@ public function __construct(Client $client, EventDispatcherInterface $eventDispa
      */
     public function display(OutputInterface $output)
     {
-        foreach ($this->packages() as $code => $progress) {
-            $output->write(sprintf('%s (%s%%)', $code, $progress), true);
+        foreach ($this->branches as $branch) {
+            $output->write(
+                sprintf("Languages exported for %s branch (%d%%):", $branch, $this->minTranslatedProgress),
+                true
+            );
+            $table = new Table($output);
+            $table->setHeaders(['locale', 'percentage']);
+            foreach ($this->packages(true, $branch) as $code => $progress) {
+                $table->addRow([$code, sprintf('%d%%', $progress)]);
+            }
+            $table->render();
         }
     }
 
     /**
      * Return the list of packages to import
      *
+     * @param bool        $exclude
+     * @param string|null $branch
+     *
      * @return array
      */
-    public function packages()
+    public function packages($exclude = true, $branch = null)
```

Comment:
there is something I don't understand here (maybe just a naming pb)
$exclude is set to true by default
which $results will always be empty by default

it seems really weird to me